src/core/server/saved_objects/service/lib/search_dsl/sorting_params.ts

```diff
@@ -38,7 +33,6 @@ export function getSortingParams(
             order: sortOrder,
           },
         },
-        ...(pit ? [ES_PROVIDED_TIEBREAKER] : []),
       ],
```

Comment:
Should we remove `pit` from the `getSortingParams` parameters, or do you foresee any reason to keep it?

Reply:
Ah great point, no I should have removed that, will update.